src/NUnitFramework/framework/Internal/TestFilter.cs

```diff
@@ -94,7 +94,7 @@ public virtual bool IsExplicitMatch(ITest test)
         /// </summary>
         /// <param name="test">The test to which the filter is applied</param>
         /// <returns>True if the filter matches the an ancestor of the test</returns>
-        private bool MatchParent(ITest test)
+        public bool MatchParent(ITest test)
```

Comment:
Because it has to be accessible from NotFilter.BaseFilter.

 

From: CharliePoole [mailto:notifications@github.com] 
Sent: Saturday, October 03, 2015 2:33 PM
To: nunit/nunit
Cc: Neil Colvin
Subject: Re: [nunit] Fix NotFilter so -exclude works correctly (#862)

 

In src/NUnitFramework/framework/Internal/TestFilter.cs <https://github.com/nunit/nunit/pull/862#discussion_r41090181> :

> @@ -94,7 +94,7 @@ public virtual bool IsExplicitMatch(ITest test)
>          /// </summary>
>          /// <param name="test">The test to which the filter is applied</param>
>          /// <returns>True if the filter matches the an ancestor of the test</returns>
> -        private bool MatchParent(ITest test)
> +        public bool MatchParent(ITest test)

Why not protected?

—
Reply to this email directly or view it on GitHub <https://github.com/nunit/nunit/pull/862/files#r41090181> .  <https://github.com/notifications/beacon/AFh64XhumHRiNQfBbJoqUndWpC9hw5Yiks5o4BbrgaJpZM4GIcgJ.gif>